### ISSUE TITLE
Integration with ecosystem: Flyway support

### ISSUE THREAD

**Issue by jseldess** [cockroachdb/docs#1563]
Background: https://airtable.com/tblD3oZPLJgGhCmch/viwdV1sM5KEK2L14K/recvRLFkfHVO4jMNH

PRs:

**Comment by jseldess** [comment]
@jordanlewis, once this work is ready for inclusion in our docs, please assign the `ready` label and link to any relevant PRs. Thanks!

**Comment by jseldess** [comment]
Closing this in favor of https://github.com/cockroachdb/docs/issues/2979, where we'll create a new "third-party tools" section and call out flyway for schema migration.
